aspnetcore/security/key-vault-configuration/samples/3.x/SampleApp/Markup.cs

```diff
@@ -0,0 +1,47 @@
+namespace SampleApp
+{
+    internal static class Markup
+    {
+        internal const string Text = @"<!DOCTYPE html>
```

Comment:
I wonder if we're going too far out of the way to not use something like Razor Pages here, but I'm not too worried either way.

Reply:
I can do that. I'll do it tomorrow. The *nice thing* about this is that it doesn't require the usual round of major RP upgrades from release-to-release. A lot of 💰 has been spent keeping samples up-to-date that don't really need to adopt RP/MVC just to make their points.

Reply:
@Rick-Anderson Do you want me to do that ... convert this to RP? It will cost more 💰 to maintain it in the future if so. I kind'a wish *many* of the samples were done the way this one was. It's been a 🐻 updating these things every major release.

Here's the evidence ... https://github.com/aspnet/AspNetCore.Docs/issues/5495

... 🙈

Reply:
Like I said, I'm not too worried either way. I think the current sample **is** clear to users, and the sample doesn't really use any Razor Pages features. It just smells a little funky to be shoving a bunch of markup in to `string.Format` when we have Razor ;).

Reply:
We could scrap all of that HTML and just have it write a string. Here's what it currently outputs for reference ...

<img width="547" alt="Capture1" src="https://user-images.githubusercontent.com/1622880/68904200-021d6a80-0703-11ea-8f02-96f4e009688b.PNG">

Reply:
A string would be a lot simpler. It wouldn't have as much ✨style✨, but it might be the best approach to avoid bringing in all of Razor Pages' baggage without a giant markup format string.

Reply:
ok ... I'll do that first thing in the morning.